.github/workflows/release.yml

```diff
@@ -32,18 +32,70 @@ jobs:
-          tag: ${{ github.event.inputs.version }}
\ No newline at end of file
+          tag: ${{ github.event.inputs.version }}
+
+      - name: Checkout crane-plugins
+        uses: actions/checkout@v2
+        with:
+          repository: ${{ github.repository_owner }}/crane-plugins
+          token: ${{ secrets.PLUGIN_RELEASE }}
+
+      - name: Updating index file and adding manifest
+        run: |
+          cat << EOF >> index.yml
+          OpenshiftPlugin-${{ github.event.inputs.version }}: https://github.com/${{ github.repository_owner }}/crane-plugins/raw/main/plugins/OpenshiftPlugin-${{ github.event.inputs.version }}.yml
+          EOF
+          cat << EOF >> plugins/OpenshiftPlugin-${{ github.event.inputs.version }}.yml
+          name: OpenshiftPlugin
+          shortDescription: OpenshiftPlugin
+          description: this is OpenshiftPlugin
+          version: ${{ github.event.inputs.version }}
+          binaries:
+            - os: linux
+              arch: amd64
+              uri: https://github.com/${{ github.repository }}/releases/download/${{ github.event.inputs.version }}/amd64-linux-openshiftplugin-${{ github.event.inputs.version }}
+            - os: darwin
+              arch: amd64
+              uri: https://github.com/${{ github.repository }}/releases/download/${{ github.event.inputs.version }}/amd64-darwin-openshiftplugin-${{ github.event.inputs.version }}
+            - os: darwin
+              arch: arm64
+              uri: https://github.com/${{ github.repository }}/releases/download/${{ github.event.inputs.version }}/arm64-darwin-openshiftplugin-${{ github.event.inputs.version }}
+          optionalFields:
+            - flagName: "strip-default-pull-secrets"
+              help:     "Whether to strip Pod and BuildConfig default pull secrets (beginning with builder/default/deployer-dockercfg-) that aren't replaced by the map param pull-secret-replacement"
+              example:  "true"
+            - flagName: "pull-secret-replacement"
+              help:     "Map of pull secrets to replace in Pods and BuildConfigs while transforming in format secret1=destsecret1,secret2=destsecret2[...]"
+              example:  "default-dockercfg-h4n7g=default-dockercfg-12345,builder-dockercfg-abcde=builder-dockercfg-12345"
+            - flagName: "registry-replacement"
+              help:     "Map of image registry paths to swap on transform, in the format original-registry1=target-registry1,original-registry2=target-registry2..."
+              example:  "docker-registry.default.svc:5000=image-registry.openshift-image-registry.svc:5000,docker.io/foo=quay.io/bar"
+          EOF
+
+      - name: Commit index and manifest file
+        run: |
+          git config --local user.email "41898282+github-actions[bot]@users.noreply.github.com"
+          git config --local user.name "github-actions[bot]"
+          git add .
+          git commit -m "Add changes" -a
+
+      - name: Push changes to crane-plugins
```

Comment:
I'm personally against committing directly to these repos but can understand the appeal.

My hope is that we can add sufficient automation to crane-plugins such that reasonable PRs can be automatically merged, making submitting a PR here (instead of direct commit) more reasonable.